lib/rule_blocklist.js

```diff
@@ -0,0 +1,84 @@
+"use strict";
+
+const a = "a"
+  , Config = require("eslint/lib/config")
+  , merge = require("eslint/lib/config/config-ops").merge
+  ;
+
+const blocklistedRules = {
+  "no-unresolved": "Code Climate doesn't fetch your project's dependencies so this rule produces high amount of false positives."
+};
```

Comment:
> ```
> Ignoring the following rules that rely on module resolution:
> 
> * import/no-unresolved
> ```

Right now we have just one reason (we know of) to turn off rules. And we have just one rule (I didn't do extensive research on this yet). It's an appropriate way to format the message for it.

However, it doesn't seem to be the only possible reason. I specifically chosen this approach because there might be others and it will be a bit easier to change it in the future — just add it to this object, no code changes needed.

Here are a few examples of other reasons we may want to disable a rule I can think of:
* A platform-specific rule that is incompatible with Linux
* A rule that relies on an external tool that is not in the container
* A rule that tries to use network
* A rule that is incompatible with Node version we run

I'm not opposing the proposed change. I'm just presenting my reasoning behind the design. I'll make the change if you prefer it.